src/polyfills/stubs/intersection-observer-stub.js

```diff
@@ -103,7 +103,12 @@ export function installStub(win) {
  */
 export function supportsDocumentRoot(win) {
   try {
-    new win.IntersectionObserver(() => {}, {root: win.document});
+    new win.IntersectionObserver(
+      () => {},
+      // TODO: closure does not yet know about root: Document.
+      // Remove this after closure support lands: https://github.com/google/closure-compiler/pull/3804.
+      /** @type ?*/ ({root: win.document})
```

Comment:
```suggestion
      /** @type {{root: Document}}*/ ({root: win.document})
```

Reply:
Closure has outdated built-in extern for InOb ctor: https://github.com/google/closure-compiler/pull/3804

Reply:
Then for now lets at least narrow it to `{{ root: ? }}`

Reply:
Done